config/initializers/formtastic.rb

```diff
@@ -24,7 +24,7 @@
   # '<abbr title="required">*</abbr>'. In other words, if you configure formtastic.required
   # in your locale, it will replace the abbr title properly. But if you don't want to use
   # abbr tag, you can simply give a string as below.
-  # Formtastic::FormBuilder.required_string = "(required)"
+  Formtastic::FormBuilder.required_string = '&nbsp;<span class="pf-c-form__label-required" aria-hidden="true">&#42;</span>'
```

Comment:
This will affect all forms in porta, right? is that OK?

Don't we have many forms including the `*` manually? woudln't this add an additional `*` to them?

Reply:
Setting this config won't add anything new but it will replace the default `<abbr title="required">*</abbr>` that is added to any input with `required: true`.
I think we only add this manually in React and Liquid templates, where we don't use formtastic.

Reply:
If you're sure it's right. OK. Approved.